addons/addon-base-raas/packages/base-raas-services/lib/cfn-templates/cfn-template-service.js

```diff
@@ -48,6 +54,19 @@ class CfnTemplateService extends Service {
 
   async getTemplate(name) {
     const entry = _.find(this.store, ['key', name]);
+    const isAppStreamEnabled = await this.settings.getBoolean(settingKeys.isAppStreamEnabled);
+    if (!isAppStreamEnabled && name === 'onboard-account' && entry) {
+      const doc = yamlParse(entry.value);
+      delete doc.Resources.AppStreamFleet;
+      delete doc.Resources.AppStreamStack;
+      delete doc.Resources.AppStreamStackFleetAssociation;
+      delete doc.Outputs.AppStreamFleet;
+      delete doc.Outputs.AppStreamStack;
```

Comment:
Rather than deleting specific Resources was it possible to remove the ones where we have condition as `isAppStream`? That way any future changes to that template (if any) would be accounted for

Reply:
updated to remove all resources/outputs with `isAppStream` or `isAppStreamAndCustomDomain` condition.